app/templates/base-flat.jade

```diff
@@ -48,6 +48,9 @@ mixin accountLinks
                 .nav-spacer
                 if !me.hideTopRightNav()
                   ul.nav.navbar-nav
+                    if me.isAnonymous() && require('app/core/experiments').getRequestAQuoteGroup(me) === 'request-a-quote-header'
```

Comment:
Will `getRequestAQuoteGroup` be available on fresh loading the website i.e. when static pages are loaded? I had added `getHomePageTestGroup` to static-mock.coffee for this reason, worth checking if its required for `getRequestAQuoteGroup`

Reply:
Sync with @shubhi1092 offline and verified this is workign.